scripts/a11y-sla.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+set -e
+
+        echo "🚀 Fetching open issues from GitHub API (handling pagination)..."
```

Comment:
For the whole file--can you remove the extra indentation? The convention is to have this be flush against the left side. Example:
```suggestion
echo "🚀 Fetching open issues from GitHub API (handling pagination)..."
```